packages/core/src/Cardano/types/Script.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-use-before-define */
+
+import { Slot } from './';
+
+export declare type ScriptPlutus = string;
+
+export interface PlutusV1 {
+  'plutus:v1': ScriptPlutus;
+}
+export interface PlutusV2 {
+  'plutus:v2': ScriptPlutus;
+}
+
+export interface ExpiresAt {
+  expiresAt: Slot;
+}
+export interface StartsAt {
+  startsAt: Slot;
+}
+
+export declare type DigestBlake2BVerificationKey = string;
```

Comment:
This could probably be better typed

Reply:
remove and replaced with the proper kind of native script, this will change once we merge the native/simple script branch and when eventually we add support for plutus v1 and v2